arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1717 @@
+            "modelAsString": "True"
+          }
+        },
+        "status": {
+          "description": "Status of the Job.",
+          "enum": ["unknown", "enqueued", "running", "completed", "failed", "cancelled"],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "JobStatus",
+            "modelAsString": "False"
+          }
+        },
+        "failureReason": {
+          "description": "If status == failure, this represents a string containing the reason.",
+          "type": "string"
+        },
+        "statusMessage": {
+          "description": "Status message for the job",
+          "type": "string"
+        },
+        "deviceId": {
+          "description": "The deviceId related to this response.",
+          "type": "string"
+        },
+        "parentJobId": {
+          "description": "The jobId of the parent job, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "IotHubDescriptionListResult": {
+      "description": "Contains the user view of a List of IotHubDescription with a next link",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of IotHubDescription",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotHubDescription"
+          }
+        },
+        "nextLink": {
+          "description": "The next link.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubQuotaMetricInfo": {
+      "description": "Contains properties related to quota metrics for an Iot Hub.",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "description": "The name of the quota metric.",
+          "type": "string"
+        },
+        "CurrentValue": {
+          "format": "int64",
+          "description": "The current value for the quota metric.",
+          "type": "integer"
+        },
+        "MaxValue": {
+          "format": "int64",
+          "description": "The maximum value of the quota metric.",
+          "type": "integer"
+        }
+      }
+    },
+    "OperationInputs": {
+      "description": "OperationInputs.",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "description": "The name of the iot hub.",
+          "type": "string"
+        }
+      }
+    },
+    "IotHubNameAvailabilityInfo": {
+      "description": "Contains the user view of properties indicating whether a given iothub name is available",
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "description": "The value indicating whether the provided name is available.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "The reason for unavailability.",
+          "enum": ["Invalid", "AlreadyExists"],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IotHubNameUnavailabilityReason",
+            "modelAsString": "False"
```

Comment:
The Travis-ci is failing because "False" should be lower case "false":

Uncaught Error: Schema validation failed: [
	{
		"code": "INVALID_TYPE",
		"params": [
			"boolean",
			"string"
		],
		"message": "Expected type boolean but found type string",
		"path": "#/definitions/IotHubNameAvailabilityInfo/properties/reason/x-ms-enum/modelAsString",
		"schemaId": "https://raw.githubusercontent.com/Azure/autorest/master/schema/swagger-extensions.json#"
	}